api/src/main/java/org/pmiops/workbench/cohortreview/mapper/CohortReviewMapper.java

```diff
@@ -28,25 +28,19 @@
   @Mapping(target = "participantCohortStatuses", ignore = true)
   CohortReview dbModelToClient(DbCohortReview dbCohortReview);
 
-  @Mapping(target = "etag", source = "version", qualifiedByName = "cdrVersionToEtag")
-  // used for pagination. Effectively deprecated, to remove with RW-4706
-  @Mapping(target = "queryResultSize", ignore = true)
-  @Mapping(target = "page", ignore = true)
-  @Mapping(target = "pageSize", ignore = true)
-  @Mapping(target = "sortOrder", ignore = true)
-  @Mapping(target = "sortColumn", ignore = true)
-  // this fetches all participants, and can be large, we don't want to fetch by
-  // default. May be removed from object pending design
-  @Mapping(target = "participantCohortStatuses", ignore = true)
-  CohortReview dbModelToClient(DbCohortReview dbCohortReview, @Context PageRequest pageRequest);
```

Comment:
No longer need this mapper method. I will create a future story that will remove page, pageSize, sortOrder and sortColumn for the CohortReview object